src/compiler/binder.ts

```diff
@@ -3495,6 +3495,9 @@ namespace ts {
             case SyntaxKind.BreakStatement:
                 transformFlags |= TransformFlags.ContainsHoistedDeclarationOrCompletion;
                 break;
+
+            case SyntaxKind.ImportCallExpression:

```

Comment:
@Kovensky That is what I was thinking about doing. treat it similar to super -> which we either parse as callExpression or propertyAccessexpression